src/theme/index.js

```diff
@@ -0,0 +1,42 @@
+const sizeMargin = {
+  s: 10,
+  xs: 20,
+  m: 30,
+  l: 40,
+}
+
+const sizeBorderRadius = {
+  s: 5,
+  xs: 10,
+  m: 20,
+  l: 40,
+}
+
+const sizePadding = {
+  s: 5,
+  xs: 10,
+  m: 20,
+  l: 40,
+}
+
+const FontSize ={
+  s: 20,
+  xs: 40,
+  m: 60,
+  l: 90,
+}
+
+export default {
```

Comment:
maybe change this to separate exports?
In situation when we need only colors we don 't want to import all theme properties